CODE_STYLE.md

```diff
@@ -15,6 +15,21 @@ tests. Some other rules are only socially enforced during code reviews.
 
 We adhere to the Style Guide for Python Code documented in the [PEP 8][2].
 
+## Multiline strings
+
+For multiline strings, we prefer to use `textwrap.dedent`:
+
+    ```
+    # end first line with \ to avoid the empty line!
+    s = textwrap.dedent('''\
```

Comment:
weren't you advocating for `"""` instead of `'''`

Reply:
updated.